src/hooks/useGetHistoryData.ts

```diff
@@ -113,9 +133,10 @@ function useGetHistoryData(
 
       const balances = getBalances();
 
+      setRawHistory(history);
```

Comment:
I see we are calling `setRawHistory` here and then `setHistoryData` on line 142 and then `setHistoryData` will also be called inside useEffect on line 106 with basically the same inputs. Why is that?